app/resources/views.py

```diff
@@ -45,18 +46,76 @@ def add():
                             user_id=int(current_user.get_id()))
         db.session.add(resource)
         db.session.commit()
-        return redirect(url_for('resources.index'))
+        return redirect(url_for('resources.show', resource_id=resource.id))
     return render_template('resources/add_resource.html', form=form)
 
 
 @resources.route('/resource/<int:resource_id>')
 @login_required
-def show_resource(resource_id):
-    """
-    Show the resource with the given id, the id is an integer.
-    """
+def show(resource_id):
```

Comment:
I was having issues with giving functions names that included underscores bc redirect(url_for('resources.show',...) https://github.com/hack4impact/women-veterans-rock/pull/26/files#diff-fcb1e97a1f2b48727b411c2ac6d366afR49 Seems like a weird issue